sdk/js/index.ts

```diff
@@ -52,9 +63,9 @@ export class Stream {
     publicClient: PublicClient,
     walletClient?: WalletClient
   ): Promise<Stream> {
-    const creator = log.topics[2] as Address;
+    const creator = ("0x" + (log.topics[2] as string).slice(-40)) as Address;
```

Comment:
Is this necessary since it's parsed as an address?

Reply:
Yes otherwise you get a couple of duplicate '0' after the 'Ox'

Reply:
oh, I mean like why does `const creator = log.topics[2] as Address` not work?

Reply:
because log topics returns this before you clean it, so it cannot be used as it is. Address type is just `0x${string}` so it doesnt count the number of characters
<img width="512" alt="Screenshot 2023-11-02 at 12 07 15" src="https://github.com/ApeWorX/ApePay/assets/117547827/853117f1-7bf9-49c5-b1f7-7946bf4bec56">